### ISSUE TITLE
Changelog doesn’t appear if it is submitted after the extension is reviewed

### ISSUE THREAD

**Issue by ExE-Boss** [mozilla/addons-server#6253]
### Describe the problem and steps to reproduce it:
1. Submit an extension update
2. Start writing the changelog for the latest version
3. Have it reviewed
4. Finish writing the changelog for the latest version
5. Submit the changelog for the latest version

### What happened?
The changelog for the version doesn’t appear

### What did you expect to happen?
The changelog for the version should appear

### Anything else we should know?
The extension and version where this happened ([Application Menu](https://addons.mozilla.org/firefox/addon/application-menu/versions/0.1.5)).

#### The version changelog should be:
```html
<b>Changes – 0.1.5</b>
<ul>
<li>Photon Theme no longer highlights the disabled expander icons on hover</li>
<li>Implemented Classic Theme for those who want the menu to resemble Firefox 4–28.</li>
<li><a href="https://addons.mozilla.org/firefox/user/ntim/">Tim Nguyen</a> implemented the Exit, Save Page As… and a part of the Fullscreen command.
<ul>
<li>Disclaimer: The Exit command might potentially cause issues with Session Restore with sufficiently large amounts of open windows, but no such issue came up during testing.</li>
</ul>
</li>
<li>The Popup is now closed when an action is performed, mimicking the behaviour of Firefox 4–28.</li>
<li>The shortcut for the Exit command was changed to Firefox’s <code>Ctrl+Shift+Q</code>.</li>
</ul>
```

Which can be seen [here](https://github.com/ExE-Boss/app-menu/releases/tag/0.1.5) on the GitHub repository.

#### Original issue:
mozilla/addons-frontend#3004

**Comment by diox** [comment]
I do see the changelog so I assume it was just a temporary caching issue...

**Comment by ExE-Boss** [comment]
Well, I still don’t see the changelog, and I uploaded the extension.

![AMO – Application Button – Missing 0.1.5 Changelog](https://user-images.githubusercontent.com/3889017/29740914-5fc74982-8a62-11e7-9ab5-06fde7079fc2.png)

**Comment by diox** [comment]
@ExE-Boss  What's the exact URL you were using (including locale - the link you give above does not include one and that redirects to an URL that is dependent on the browser configuration) to reproduce the issue?

**Comment by ExE-Boss** [comment]
The full URL is [https://addons.mozilla.org/en-GB/firefox/addon/application-menu/versions/0.1.5](https://addons.mozilla.org/en-GB/firefox/addon/application-menu/versions/0.1.5).

It works on en-US, but not on en-GB.

**Comment by diox** [comment]
Yeah, that's it. Somehow you ended up with an empty (but existing) translation for your release notes in `en-GB`, and that's causing the issue. This is really a devhub issue, we need to figure out how that happened.

**Comment by ExE-Boss** [comment]
I can confirm that this has also happened with my [about:about Button](https://addons.mozilla.org/en-GB/firefox/addon/firefox-about-button/versions/0.1.16) extension.

**Comment by ExE-Boss** [comment]
It seems to be fixed now for [Application Menu 0.1.5](https://addons.mozilla.org/en-GB/firefox/addon/application-menu/versions/0.1.5) and [about:about Button 0.1.17](https://addons.mozilla.org/en-GB/firefox/addon/firefox-about-button/versions/0.1.17), but not for [about:about Button 0.1.16](https://addons.mozilla.org/en-GB/firefox/addon/firefox-about-button/versions/0.1.16).
